packages/postcss/README.md

```diff
@@ -41,6 +41,9 @@ module.exports = {
 
 ## Options
 
+#### minimize *(optional)*
+Enable CSS minification.
```

Comment:
I think it should say that it's handled by the `css-loader`.

Reply:
Also need to add this options to sass block readme as well.

Reply:
Good points — done ;-)